docs/metrics/prometheus/list.md

```diff
@@ -4,113 +4,123 @@ Each metric includes a label for the server that calculated the metric. Each met
+| `minio_s3_requests_5xx_errors_total`          | Total number S3 requests with (5xx) errors.                                                                     |
+| `minio_s3_requests_canceled_total`            | Total number S3 requests canceled by the client.                                                                |
+| `minio_s3_requests_errors_total`              | Total number S3 requests with (4xx and 5xx) errors.                                                             |
+| `minio_s3_requests_incoming_total`            | Volatile number of total incoming S3 requests.                                                                  |
+| `minio_s3_requests_inflight_total`            | Total number of S3 requests currently in flight.                                                                |
+| `minio_s3_requests_rejected_auth_total`       | Total number S3 requests rejected for auth failure.                                                             |
+| `minio_s3_requests_rejected_header_total`     | Total number S3 requests rejected for invalid header.                                                           |
+| `minio_s3_requests_rejected_invalid_total`    | Total number S3 invalid requests.                                                                               |
+| `minio_s3_requests_rejected_timestamp_total`  | Total number S3 requests rejected for invalid timestamp.                                                        |
+| `minio_s3_requests_total`                     | Total number S3 requests.                                                                                       |
+| `minio_s3_requests_waiting_total`             | Number of S3 requests in the waiting queue.                                                                     |
+| `minio_s3_requests_ttfb_seconds_distribution` | Distribution of the time to first byte across API calls.                                                        |
+| `minio_s3_traffic_received_bytes`             | Total number of s3 bytes received.                                                                              |
+| `minio_s3_traffic_sent_bytes`                 | Total number of s3 bytes sent.                                                                                  |
+| `minio_software_commit_info`                  | Git commit hash for the MinIO release.                                                                          |
+| `minio_software_version_info`                 | MinIO Release tag for the server.                                                                               |
+| `minio_usage_last_activity_nano_seconds`      | Time elapsed (in nano seconds) since last scan activity.                                                        |
 
 
+# List of metrics exported per bucket level
```

Comment:
For some of the metrics like 
```
minio_bucket_requests_4xx_errors_total
minio_bucket_requests_inflight_total
minio_bucket_requests_total
minio_bucket_requests_ttfb_seconds_distribution
minio_bucket_traffic_sent_bytes
```
other than existing buckets in the instance, I see metrics for two additional buckets with names `api` and `probe-bucket-sign-xicgc15djbyz`. 
Are these inbuilt buckets? If so, do we need these metrics as well to be exposed/listed?

Reply:
This could be because some application is sending non-existent bucket names, so not an issue.

Reply:
yes we capture all those buckets as well which come in @shtripat

Reply:
it is added to ensure that we capture whatever hits MinIO at bucket resource.